src/time_integration/methods_2N.jl

```diff
@@ -42,9 +48,14 @@ struct CarpenterKennedy2N54 <: SimpleAlgorithm2N
 end
 
 """
-      CarpenterKennedy2N43()
+        CarpenterKennedy2N43()
```

Comment:
```suggestion
    CarpenterKennedy2N43()
```
Shouldn't it be 4 spaces?

Reply:
[Here](https://trixi-framework.github.io/Trixi.jl/stable/reference-trixi/#Trixi.CarpenterKennedy2N43) we can see that currently there are two additional spaces.